### PR TITLE
auto-start autoproj watch in new workspaces

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -52,7 +52,12 @@ function setupEvents(rockContext: context.Context, extensionContext: vscode.Exte
                 handleNewWorkspaceFolder(folder.uri.fsPath, rockContext, workspaces, configManager);
             });
             event.removed.forEach((folder) => {
-                workspaces.deleteFolder(folder.uri.fsPath);
+                let deletedWs = workspaces.deleteFolder(folder.uri.fsPath);
+                if (deletedWs) {
+                    deletedWs.readWatchPID().
+                        then((pid) => process.kill(pid)).
```
Comment (g-arjones) [inline src/extension.ts]: This defaults to SIGTERM.. Wouldn't it be better to send a SIGINT instead?

**Comment by g-arjones** [comment]
Cool :+1:

File: src/extension.ts (hunk 1 above)
Comment (doudou) [reply src/extension.ts]: Good catch, I'll update

**Comment by doudou** [comment]
Updated

**Pull request merged**